static/build.html

```diff
@@ -2,32 +2,34 @@
 <html lang="en" prefix="og: http://ogp.me/ns#">
     <head>
         <meta charset="utf-8"/>
-        <title>Build | GrapheneOS</title>
-        <meta name="description" content="Building instructions for GrapheneOS, a security and privacy focused mobile OS with Android app compatibility."/>
+        <title>GrapheneOS</title>
+        <meta name="description" content="GrapheneOS is a security and privacy focused mobile OS with Android app compatibility."/>
         <meta name="theme-color" content="#212121"/>
         <meta name="msapplication-TileColor" content="#ffffff"/>
         <meta name="viewport" content="width=device-width, initial-scale=1"/>
         <meta name="twitter:site" content="@GrapheneOS"/>
         <meta name="twitter:creator" content="@GrapheneOS"/>
-        <meta property="og:title" content="GrapheneOS build documentation"/>
-        <meta property="og:description" content="Building instructions for GrapheneOS, a security and privacy focused mobile OS with Android app compatibility."/>
+        <meta property="og:title" content="GrapheneOS"/>
+        <meta property="og:description" content="GrapheneOS is a security and privacy focused mobile OS with Android app compatibility."/>
         <meta property="og:type" content="website"/>
         <meta property="og:image" content="https://grapheneos.org/opengraph.png"/>
         <meta property="og:image:width" content="512"/>
         <meta property="og:image:height" content="512"/>
         <meta property="og:image:alt" content="GrapheneOS logo"/>
-        <meta property="og:url" content="https://grapheneos.org/build"/>
+        <meta property="og:url" content="https://grapheneos.org/"/>
         <meta property="og:site_name" content="GrapheneOS"/>
         <link rel="icon" type="image/vnd.microsoft.icon" href="/favicon.ico"/>
         <link rel="mask-icon" href="/mask-icon.svg" color="#1a1a1a"/>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
I don't want to include any external resources. Everything needs to be served as first-party content.

Reply:
My apologies, I assumed the material lite framework was a framework that wanted to be used based on the comments in the grapheneos.css file, however I can now understand that it is just the theme and not the actual framework.